packages/discount-code-exporter/src/main.js

```diff
@@ -153,17 +153,30 @@ export default class DiscountCodeExport {
   }
 
   // Use this method to make the `cartDiscounts`
-  // field compatible with  the importer
+  // field compatible with the importer
   _processCode (data: CodeData): Object {
-    const cartDiscounts = data.cartDiscounts.reduce((
+    const newCodeObj = data
```

Comment:
Why do you create a new variable `newCodeObj `?
`_processCode (newCodeObj: CodeData): Object {` Should be enought?